helmfile.d/values/prometheus-alerts-sc.yaml.gotmpl

```
@@ -35,6 +35,7 @@ defaultRules:
     networkpolicies: {{ .Values.networkPolicies.enableAlerting }}
     clusterApi: {{ .Values.clusterApi.enabled }}
     webhooks: {{ .Values.prometheus.webhookAlerts.enabled }}
+    openstack: {{ or (.Values | get "openstackMonitoring.enabled" false) (.Values | get "clusterApi.enabled") false }}
```

Shouldn't it be like this?
```suggestion
    openstack: {{ or (.Values | get "openstackMonitoring.enabled" false) (.Values | get "clusterApi.enabled" false) }}
```